docs/faq.rst

```diff
@@ -0,0 +1,115 @@
+Frequently Asked Questions
+==========================
+
+The secondary is blocked in the CATCHING_UP state, what should I do?
+--------------------------------------------------------------------
+
+In the pg_auto_failover design, the following two things are needed for the
+monitor to be able to orchestrate nodes integration completely:
+
+ 1. Health Checks must be successful
```

Comment:
We have this pattern all over, the bullets are highlighted with gray, which feels unintuitive to me as this colur is associated with `old` comments in github. This is probably a personal taste, do not require changes, but wanted to note anyway.

<img width="1288" alt="Screen Shot 2020-09-01 at 13 36 28" src="https://user-images.githubusercontent.com/7829470/91849020-aef9b180-ec5b-11ea-9406-8cae0b7f6560.png">

Reply:
Sphinx renders this without this (also shows the result of the `:ref:` thing):
![image](https://user-images.githubusercontent.com/1162278/91850579-0e58c100-ec5e-11ea-913f-8515038ec84a.png)